site/source/tutorials/002_creating_an_application/index.md

```diff
@@ -28,58 +28,62 @@ You also need to be familiar with TypeScript as Dojo uses it extensively.
 
 The second image shows how widgets ensure that components only interact according to their design intent. The widget encapsulates its visual and behavioral aspects. It then exposes properties and methods that allow other components to interact with it. By providing a controlled interface, it is much easier to keep the visual and behavioral aspects of the widget synchronized.
 
-In our demo application, we only have one widget, the HelloWorld widget:
+In our demo application, we don't currently have any widgets, but let's abstract the virtual DOM from `main.ts` out into a dedicated widget. You'll notice that to render a widget, we use a different pragma, `w()`, than we used before to render nodes that directly represent the DOM nodes.
+
+{% instruction 'Create a widget `HelloWorld.ts` in `src/widgets`' %}
+
+{% include_codefile 'demo/finished/biz-e-corp/src/widgets/HelloWorld.ts' %}
+
+{% instruction 'Use the `HelloWorld` widget in the `renderer` in `src/main.ts`' %}
 
-{% include_codefile 'demo/initial/biz-e-corp/src/widgets/HelloWorld.ts' %}
+{% include_codefile 'demo/finished/biz-e-corp/src/main.ts' lines:5 %}
 
-This is very simple, containing a single `<h1>` tag and no behavior, but it demonstrates some important concepts. Notice the `render` method, which provides the virtual nodes (also known as DNodes) for the Projector to determine what to add to the HTML document. In this example, the widget is simple enough that the function always returns the same result. We could make this widget more sophisticated by allowing it to take additional properties that can be used to alter how the DNodes are generated. We can provide sensible default values for these properties to add more complex behavior without having to change how the rest of the application interacts with it. This encourages the development of loosely coupled components that are easier to develop and maintain over time.
+This is very simple widget, containing the virtual DOM that we previously were returning straight from the vdom `renderer`, but it demonstrates some important concepts. Notice the `render` method, which provides the virtual nodes (also known as DNodes) for the `renderer` to determine what to add to the HTML document. In this example, the widget is simple enough that the function always returns the same result. We could make this widget more sophisticated by allowing it to take additional properties that can be used to alter how the DNodes are generated. We can provide sensible default values for these properties to add more complex behavior without having to change how the rest of the application interacts with it. This encourages the development of loosely coupled components that are easier to develop and maintain over time.
```

Comment:
This is very simple widget -> This is a very simple widget

without having to change how the rest of the application interacts with it. -> without needing to change how the rest of the application interacts with the widget.

This encourages the development -> This approach encourages the development